en/_posts/2023-09-29-groonga-13.0.8.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post.en
+title: Groonga 13.0.8 has been released
+description: Groonga 13.0.8 has been released!
+published: false
+---
+
+## Groonga 13.0.8 has been released
+
+[Groonga 13.0.8](/docs/news.html#release-13-0-8) has been released!
+
+How to install: [Install](/docs/install.html)
+
+### Changes
+
+Here are important changes in this release:
+
+### Improvements
+
+* [[column_create](/docs/reference/commands/column_create.html)] Improved `column_create` flags with addition new flags `COLUMN_FILTER_SHUFFLE`, `COLUMN_FILTER_BYTE_DELTA`, `COMPRESS_FILTER_TRUNCATE_PRECISION_1BYTE`, and `COMPRESS_FILTER_TRUNCATE_PRECISION_2BYTES`.
+
+* Added new bundling library Blosc.
+
+  `COLUMN_FILTER_SHUFFLE`, `COLUMN_FILTER_BYTE_DELTA`, `COMPRESS_FILTER_TRUNCATE_PRECISION_1BYTE`, and `COMPRESS_FILTER_TRUNCATE_PRECISION_2BYTES` flags are require Blosc.
+
+* [[status](/docs/reference/commands/status.html)] Improved `status` output with addition new features `"blosc"`.
+
+* [[groonga](/docs/reference/executables/groonga.html)] Improved `groonga --version` output with addition new value `blosc`.
+
+* [[select](/docs/reference/commands/select.html)] Improved `select` arguments with addition new argument `--fuzzy_max_distance` (experimental).
+
+* [[select](/docs/reference/commands/select.html)] Improved `select` arguments with addition new argument `--fuzzy_max_expansions` (experimental).
+
+* [[select](/docs/reference/commands/select.html)] Improved `select` arguments with addition new argument `--fuzzy_max_distance_ratio` (experimental).
```

Comment:
```suggestion
* [[select](/docs/reference/commands/select.html)] Improved `select` arguments with additional new argument `--fuzzy_max_distance_ratio` (experimental).

  This argument is useful when search process can tolerance typographical errors. This argument revise typographical errors by users.

In case of setting this argument, Groonga proceed search again after checking a word with edit distance indicated under `--fuzzy_max_distance` from the vocabulary list when the query does not match as shown in following example.

table_create Memos TABLE_NO_KEY
column_create Memos content COLUMN_SCALAR ShortText

table_create Lexicon TABLE_PAT_KEY ShortText   --default_tokenizer TokenNgram   --normalizer NormalizerNFKC150
column_create Lexicon memos_content   COLUMN_INDEX|WITH_POSITION Memos content
load --table Memos
[
{"content": "This is a pen"},
{"content": "That is a pen"}
]

select Memos   --match_columns content   --query "Thjs"   --fuzzy_max_distance 1   --output_columns *,_score
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "content",
          "ShortText"
        ],
        [
          "_score",
          "Int32"
        ]
      ],
      [
        "This is a pen",
        1
      ]
    ]
  ]
]
```